### ISSUE TITLE
Add support of visdom

### ISSUE THREAD

**Issue by sivanWu0222** [victoresque/pytorch-template#64]
> e.g. use visdom to visualize the process of training loss.

**Comment by SunQpark** [comment]
Thank you for issuing this, @sivanWu0222.
Can you give me more information, like what feature of visdom do you want to have in this project?
Does it have some nice functionalities that tensorboard doesn't have?
Do you know some good reference repositories using visdom well?
